src/modules/commander/Commander.cpp

```diff
@@ -294,8 +294,12 @@ int Commander::custom_command(int argc, char *argv[])
 		vehicle_control_mode_s vehicle_control_mode{};
 		vehicle_control_mode_sub.copy(&vehicle_control_mode);
 
+		uORB::Subscription failure_detector_status_sub{ORB_ID(failure_detector_status)};
```

Comment:
Next we should either kill "check" entirely (once it's easy enough to get all the details directly) or replace the direct call with MAV_CMD_RUN_PREARM_CHECKS. I think I had this in that test/hack branch.

Reply:
Right, there should be no static context call to the preflight checks. In fact, there should only be one periodical and one messaging call until we have events.